doc/interfaces-vrrp.scd

```diff
@@ -0,0 +1,199 @@
+interfaces-vrrp(5)
```

Comment:
This file starts with line breaks at a certain column, then trails off to a wider column, then appears to have no determinate column for line breaks at all.

Reply:
Ok, I can fix this.

Reply:
> This file starts with line breaks at a certain column, then trails off to a wider column, then appears to have no determinate column for line breaks at all.

Can you explain me better what do you mean with this? I was looking to my file and this file `interfaces-vxlan.scd` and I don't see any differences. My file is based on `interfaces-vxlan.scd`.